salt/_modules/metalk8s_solutions.py

```diff
@@ -324,6 +324,11 @@ def manifest_from_iso(path):
             )
         )
 
+    if manifest is None:
+        raise CommandExecutionError(
+            "No such {} file found in {}".format(SOLUTION_MANIFEST, path)
```

Comment:
Error message could be more "imperative": `"Solution ISO at {} must contain a '{}' file".format(...)`